### ISSUE TITLE
font weight medium not bold enoughtl

### ISSUE THREAD

**Issue by Lucascoorek** [tokotype/PlusJakartaSans#12]
When loading medium font to a browser throught css, medium font in not bold enoughtl.
CSS:

```
@font-face {
    font-family: PlusJakartaSans;
    font-weight: 400;
    src: url("/fonts/PlusJakartaSans-Regular.woff2") format("woff2");
}
@font-face {
    font-family: PlusJakartaSans;
    font-weight: 700;
    src: url("/fonts/PlusJakartaSans-Bold.woff2") format("woff2");
}
@font-face {
    font-family: PlusJakartaSans;
    font-weight: 500;
    src: url("/fonts/PlusJakartaSans-Medium.woff2") format("woff2");
}

```

This is how it looks in the browser:
![Screenshot 2021-02-22 at 10 57 23](https://user-images.githubusercontent.com/47325777/108692426-e0618480-74fc-11eb-9a3b-2bb94ef98050.jpg)

Medium font is almost identical to regular.

**Comment by kenmcd** [comment]
It looks right based on the weights.
Regular (400), Medium (500), Bold (700)
What is missing is the SemiBold (600).
When SemiBold (600) is added it looks like this:

![Plus Jakarta Sans with SemiBold added](https://user-images.githubusercontent.com/844920/108768845-0c850180-750d-11eb-9347-866c742dc7e1.png)

I think that may be what you are looking for.
Not sure why there is no SemiBold as it is very easy to add (like I did in just minutes).
Perhaps you should re-title this post as a feature request for a SemiBold (600) weight.

**Comment by amoghsrivastava** [comment]
@kenmcd How did you get the above weights? Could you please share this? 😃

**Comment by amoghsrivastava** [comment]
> It looks right based on the weights.
> Regular (400), Medium (500), Bold (700)
> What is missing is the SemiBold (600).
> When SemiBold (600) is added it looks like this:
> 
> ![Plus Jakarta Sans with SemiBold added](https://user-images.githubusercontent.com/844920/108768845-0c850180-750d-11eb-9347-866c742dc7e1.png)
> 
> I think that may be what you are looking for.
> Not sure why there is no SemiBold as it is very easy to add (like I did in just minutes).
> Perhaps you should re-title this post as a feature request for a SemiBold (600) weight.

@kenmcd How did you get the above weights? Could you please share this? 😃

**Comment by kenmcd** [comment]
> @kenmcd How did you get the above weights? Could you please share this?

I just added an instance in the .glyphs file.
Don't remember if I even saved it.
I can take a look tomorrow. 
What are you looking for - TTF, WOFF2 ?

**Comment by amoghsrivastava** [comment]
> > @kenmcd How did you get the above weights? Could you please share this?
> 
> I just added an instance in the .glyphs file.
> Don't remember if I even saved it.
> I can take a look tomorrow.
> What are you looking for - TTF, WOFF2 ?

Ahh okay, I tried doing that but it didn’t work.. I’m looking for TTFs

**Comment by mhoev** [comment]
@kenmcd 
Hi, this font is awesome!
Would it be possible to share the SemiBold also in WOFF2 and WOFF?

**Comment by kenmcd** [comment]
> 
> 
> @kenmcd Hi, this font is awesome! Would it be possible to share the SemiBold also in WOFF2 and WOFF?

I agree. These are really nice fonts. Just need a SemiBold weight added :-) 

Here ya go: [PlusJakartaSans-SemiBold.Reg.and.Ital.WOFF.and.WOFF2.zip](https://github.com/tokotype/PlusJakartaSans/files/7254365/PlusJakartaSans-SemiBold.Reg.and.Ital.WOFF.and.WOFF2.zip)

Not extensively tested. 
Just checked names, and viewed an HTML page test.
Let me know if there are any issues.
